modules/odc-creating-vms.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="odc-creating-vms_{context}"]
+= Using the Developer perspective to create a virtual machine
+
+You can create a virtual machine by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* A cluster administrator has installed the {VirtProductName} Operator in your cluster.
+* You have logged in to the {product-title} web console and are in the *Developer* perspective.
+
+.Procedure
+
+. In the *+Add* view, click on the *Virtual Machines* tile to see the *Create new VirtualMachine* page.
+
+. Select a virtual machine type from the *Template catalog*.
+
+. Click *Quick create VirtualMachine* to quick create a virtual machine, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
```

Comment:
```suggestion
. Click *Quick create VirtualMachine* to create a virtual machine with no further input, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
```
- Not sure if this changes the meaning. Or maybe add `to create a virtual machine immediately`. Anyway, I don't think it should be the exact phrase from the button/link

Reply:
nice suggestion, thanks